src/quo/components/text_input.cljs

```diff
@@ -178,8 +178,9 @@
                             (and platform/ios? (= keyboard-type "visible-password"))
                             "default"
 
-                            (and platform/android? secure-text-entry @visible)
-                            "visible-password"
+                            ;; NOTE: Now switching dynamically brakes android input
+                            ;; (and platform/android? secure-text-entry @visible)
+                            ;; "visible-password"
```

Comment:
in this case, it might be saved by OS and used in suggestions like regular text ?

Reply:
Yes, it will be a regular text. Not sure how OS stores it

Reply:
Found another fix, will push soon

Reply:
done